### ISSUE TITLE
Styled components not working in IE11 - 10

### ISSUE THREAD

**Issue by raghvendradeshpande** [styled-components/styled-components#1726]
Styled components are not working in IE11 - 10 with ReactJS application.  React Version is 16.3.0

**Comment by probablyup** [comment]
Is there a console error?

**Comment by raghvendradeshpande** [comment]
There is no console error. Styled components are not working on IE browser. It is working fine on Chrome and Firefox browser. Please let me know if you need more information. thank you

**Comment by probablyup** [comment]
Yeah this isn't enough information to go off of, sorry. Please follow the issue template and try providing the data requested in it.

We run styled-components with IE 11 and it works fine, so I suspect it's something to do with your environment.

**Comment by probablyup** [comment]
Hmm technically IE10 & 11 did have an earlier spec of vendor prefixed grid
layout. I wonder if stylis doesn’t handle it or if the specs diverged too
much.

**Comment by tronin** [comment]
Hi, guys we have the same issue.
You can reproduce that using this tools:
https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/
![image](https://user-images.githubusercontent.com/4414123/43201197-751d735a-9020-11e8-864c-b10a143d39f4.png)

**Comment by kitten** [comment]
@tronin please open a new issue. Since this one doesn’t even have any output / description / reproduction it’s strictly speaking not the same.

It’s also quite time consuming for us to just set up a whole VM environment, write some styled-components code, and then hope to run into any issue.

Hope that makes sense. Cheers! 🙌
